src/Workspaces/MSBuildTest/Resources/ProjectFiles/CSharp/AdditionalFile.csproj

```diff
@@ -12,7 +12,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>CSharpProject</RootNamespace>
     <AssemblyName>CSharpProject</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+   <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
Indentation of all the TargetFrameworkVersion properties in these test projects are now off by a space.